client/components/TestQueue/index.jsx

```diff
@@ -153,7 +153,7 @@ class TestQueue extends Component {
 }
 
 TestQueue.propTypes = {
-    admin: PropTypes.number,
+    admin: PropTypes.bool,
```

Comment:
I had to fix this in https://github.com/bocoup/aria-at-app/pull/88 to pass lint & tests

Reply:
whoops, sorry, I'll rebase.